python/lsst/validate/drp/validate.py

```diff
@@ -200,9 +206,25 @@ def run(repo, visitDataIds, good_mag_limit=21.0,
         Expected photometric RMS [mmag] across visits.
     matchRef : int, optional
         Expectation of the number of stars that should be matched across visits.
+    makePrint : bool, optional
+        Print calculated quantities (to stdout).
+    makePlot : bool, optional
+        Create plots for metrics.  Saved to current working directory.
+    makeJson : bool, optional
+        Create JSON output file for metrics.  Saved to current working directory.
+    outputPrefix : str
+        Specify the beginning filename for output files.
+
+    Outputs
```

Comment:
@jonathansick can correct me but I think `Outputs` is the wrong section name here.